src/components/views/read/cvForm/PersonalInfo.tsx

```diff
@@ -0,0 +1,350 @@
+import dayjs, { Dayjs } from 'dayjs';
+
+import { FC, RefObject, useRef, useState } from 'react';
+import PhoneInput from 'react-phone-input-2';
+import 'react-phone-input-2/lib/style.css';
+
+import NavigateBeforeIcon from '@mui/icons-material/NavigateBefore';
+import NavigateNextIcon from '@mui/icons-material/NavigateNext';
+import UploadFileIcon from '@mui/icons-material/UploadFile';
+import { Button, MenuItem, Select, TextField } from '@mui/material';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { DatePicker } from '@mui/x-date-pickers/DatePicker';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+
+interface InnerObject {
+  [key: string]: string;
+}
+
+interface ValuesObject {
+  [key: string]: InnerObject;
+}
+
+interface HandleModifyFunction {
+  (category: string, modifiedValues: any): void;
+}
+interface Props {
+  nextPage: () => void;
+  prevPage: () => void;
+  nextStep: () => void;
+  prevStep: () => void;
+  values: ValuesObject;
+  handleValues: HandleModifyFunction;
+}
+
+const PersonalInfo: FC<Props> = ({
+  nextPage,
+  prevPage,
+  nextStep,
+  prevStep,
+  values,
```

Comment:
In this component you only handle `values[0]`, so you should probably only pass `values[0]` as a prop, which would simplify your handling inside.

Reply:
EDIT: I understand that the values you pass "should be the specific array containing the cards info for this specific component", but his is not the case, you are passing in the full object of steps: `{"Personal Info": [{"First Name": ... }], "Education": [{...}, {...}]}`